components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -1327,4 +1333,31 @@ - (void)testSetAccessibilityTraitsIncludesButtonExplicitly {
                  UIAccessibilityTraitAllowsDirectInteraction | UIAccessibilityTraitButton);
 }
 
+- (void)testShadowColorRespondsToDynamicColor {
+  if (@available(iOS 13.0, *)) {
```

Comment:
does this need to be wrapped with `#if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)`

Reply:
Yes, it seems so.

> components/Buttons/tests/unit/ButtonsTests.m:1342:18: error: no known class method for selector 'colorWithDynamicProvider:'
        [UIColor colorWithDynamicProvider:^(UITraitCollection *traitCollection) {

Reply:
Done